commit 2b2f05fea3df031941f1bb7f2bd6728fc0775282
Author: Kefu Chai <kchai@redhat.com>
Date:   Wed Aug 15 20:31:36 2018 +0800

    auth: switch to LockPolicy templatized alternatives
    
    this change
    
    * is the first step to use std::mutex, etc directly instead our Mutex
      wrappers for better performance. see
      https://trello.com/c/aUSezBRH/365-stdmutex-etc-for-release-builds
    * pave the road to lockless auth/ subsystem in seastar.
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index ebdacaf6901..702bf1629b5 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -188,8 +188,7 @@ bool DaemonServer::ms_verify_authorizer(
   s->inst.addr = con->get_peer_addr();
   AuthCapsInfo caps_info;
 
-  RotatingKeyRing *keys = monc->rotating_secrets.get();
-  if (keys) {
+  if (auto keys = monc->rotating_secrets.get(); keys) {
     is_valid = handler->verify_authorizer(
       cct, keys,
       authorizer_data,
